src/main/java/org/folio/service/mapping/processor/translations/TranslationsHolder.java

```diff
@@ -1,16 +1,44 @@
 package org.folio.service.mapping.processor.translations;
 
 import io.vertx.core.json.JsonObject;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.service.mapping.settings.Settings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+import java.util.Optional;
 
 public enum TranslationsHolder implements TranslationFunction {
+
   SET_VALUE() {
     @Override
     public String apply(String value, JsonObject parameters, Settings settings) {
       return parameters.getString("value");
     }
+  },
+  SET_NATURE_OF_CONTENT_TERM() {
+    @Override
+    public String apply(String id, JsonObject parameters, Settings settings) {
+      Optional<JsonObject> optionalEntry = settings.getNatureOfContentTerms().stream()
```

Comment:
Probably makes sense to use a Map to store the settings with Id as the key , instead of a list, instead of traversing it each time, map gives you a constant time operation
https://github.com/folio-org/mod-data-export/pull/60/files#diff-f16bbac4236a4ff04032f19d7b0776fcR43